mr/src/test/java/org/apache/iceberg/mr/hive/serde/objectinspector/TestIcebergRecordObjectInspector.java

```diff
@@ -61,4 +61,17 @@ public void testIcebergRecordObjectInspector() {
     Assert.assertEquals(innerRecord.get(0), innerSoi.getStructFieldData(innerData, stringField));
   }
 
+  @Test
+  public void testIcebergRecordObjectInspectorWithRowNull() {
+    Schema schema = new Schema(
+            required(1, "integer_field", Types.IntegerType.get()),
+            required(2, "struct_field", Types.StructType.of(
+                    Types.NestedField.required(3, "string_field", Types.StringType.get())))
+    );
```

Comment:
Nit: The indenation is off here.

There should be two spaces when the indentation changes, and 4 spaces when the same code statement / expression continues on the next line. Like so. 
```java
    Schema schema = new Schema(
        required(1, "integer_field", Types.IntegerType.get()),
        required(2, "struct_field", Types.StructType.of(
            Types.NestedField.required(3, "string_field", Types.StringType.get())))
    );
```

Reply:
fix